### DISCUSSION TITLE
Problem with steel texture from X-ray diffraction

### DISCUSSION THREAD

**Discussion by MartinL75** [mtex-toolbox/mtex#2111] [Ask Anything] [answered]
Hi,
I'm trying to analyze a texture from a cold-rolled, low-carbon steel sheet. (110), (200) and (211) pole figures were measured with a Philips X'pert x-ray diffractometer (transmission geometry). Reading, correction and ODF calculation were done with MTEX v.5.1.1. (Script attached below as txt file.)
[AcLF_AR.txt](https://github.com/mtex-toolbox/mtex/files/14894647/AcLF_AR.txt)

These are the raw pole figures plotted from the .rw1 files:

![AcLF-AR_rawPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/e0f3c8ee-97e5-4b5c-8bac-65b77fb37974)
X=TD; Y=RD; Z=ND (i.e., transverse, rolling and normal directiones of the sheet metal).

By using defocussing and background measurementes I obtained the following corrected PFs:

![AcLF-AR_correctedPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/c6b18039-b6f7-4619-9c24-febc5240acb1)

Finally, from the calculated ODF the recalculated PFs look like this:

![AcLF-AR_reconstrPFs](https://github.com/mtex-toolbox/mtex/assets/74614118/e3f3fe69-3b33-4072-b85e-add9490926e4)

The high intensity of the (200) poles aligned to the ND appears to be artificial, since almost no poles appear in the corresponding raw PF. Furthermore, this texture does not coincide with the tipical cold-rolling, ferrictic steel textures found in the literature.
Could be something wrong in the ODF reconstruction method? 

Any information would be very helpful. Thanks in advance.

Martín Leonard
IFIR - CONICET/UNR
Rosario, Argentina.

**Comment by ralfHielscher** [comment]
Is it possible for you to share the data as well?

Ralf.

**Reply by MartinL75** [reply]
Hi Ralph,
Thanks for the quick answer.
Yes, please find the zip file attached.
[RW1_Ac-LF_AR.zip](https://github.com/mtex-toolbox/mtex/files/14897368/RW1_Ac-LF_AR.zip)
Regards

**Comment by ralfHielscher** [comment, marked as answer]
Hi Martin,

The point seems to be, that the first pole figure has so much stronger intensities in comparison to the other pole figures. If you do
```matlab
odf = calcODF(pfc.normalize,'halfwidth',5*degree)
```
you will end up with the following result
![pf](https://github.com/mtex-toolbox/mtex/assets/8619241/02b8fa8b-1387-4e83-8b5e-0094cac343eb)

The normalization ensures that the intensities are comparable between the pole figures.

Ralf.

**Reply by MartinL75** [reply]
Thank you very much! I forgot that normalization was necessary before calculating the ODF.
Regards

**Reply by ralfHielscher** [reply]
I Martin,

would it be possible to reuse your example in the MTEX documentation in order to illustrate the effect of normalization.

Ralf.

**Comment by MartinL75** [comment]
Hi Ralph, yes, of course. I'd be happy if the case results useful for the community.
Thanks.

**Reply by ralfHielscher** [reply]
Hi Martin,

thank you very much. Could you please write some lines about your background / defocussing correction. I guess the y values in your file are from a powder sample. I wondering how you can access both background and defocussing from it.

Best,

Ralf.

**Reply by ralfHielscher** [reply]
Also the powder intensities in your file are the same for 110 and 200. Is there a typo?

**Reply by MartinL75** [reply]
Hi Ralph.
Actually, the data for the background/defocussing correction come from a simpler way. We run a XRD scan at a given 2theta angle, near the tail of one of the diffraction peaks (considering that the background intensity is represented there), and tilt the sample so as to record the defocussing intensity of that background. The result is an intensity vector of 18 values, each one corresponding to 0, 5, 10, ... 80 and 85 deg. of the polar angle. That's why the correction vectors (and corresponding PFs) are treated the way that the script  shows.
Depending on the background seen at the tails of the peaks, we either make one scan per peak, or use the correction scan for two peaks. That's why the 110 and 200 intensities are the same in the file, as yo noticed.
Of course this is not the rigourous way, but from previous experience in our Lab (i.e., comparing with the powder samples) it turned out enough for our regular texture measurements.

**Reply by MartinL75** [reply]
Turns out that I'm facing a similar issue with the texture of an aluminum sheet. I'm starting a new Discussion connected to this one, if it's OK for you. Thanks
